envi/archs/msp430/__init__.py

```diff
@@ -24,11 +24,9 @@ def archGetRegCtx(self):
     def archGetNopInstr(self):
         return b'\x03\x43' # NOP is emulated with: MOV #0, R3
 
-    def archGetRegisterGroups(self):
-        groups = envi.ArchitectureModule.archGetRegisterGroups(self)
-        general= ('general', registers, )
-        groups.append(general)
-        return groups
+    def initRegGroups(self):
```

Comment:
apparently MSP430 didn't get updated with the change to Dict-based register groups